src/theme/components/header/headerSearchForm.ts

```diff
@@ -26,16 +21,17 @@ const HeaderSearchForm = {
       maxWidth: { md: "1312px" },
       whiteSpace: "initial",
       "> div": {
+        margin: "20px 15px 0",
         marginLeft: { md: "m", lg: "140px" },
         marginRight: { md: "m" },
       },
     },
-    fieldset: {
-      margin: { base: "20px 15px 0", md: "0" },
+    "#radio-group-search-type": {
+      margin: { base: "0 20px 20px", md: "0" },
     },
     textInput: {
       label: {
-        fontSize: { md: "18px" },
+        fontSize: { base: "heading.tertiary", md: "heading.secondary" },
```

Comment:
When viewing at the smallest breakpoint (320px), the input label wraps oddly.  I don't think we are going to get a lot of users with that screen size, but let's change `fontSize` to the following to make an accommodation:
```
fontSize: { base: "heading.callout", sm: "heading.tertiary", md: "heading.secondary" },
```